KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -145,7 +145,7 @@ public enum UITexts {
 
     /* Demand API (Karhoo specific) errors */
     public enum KarhooError {
-        public static let K3002 = "Text.KarhooError.K3002".localized // no availability in requested area
+        public static let K3002 = "K3002".localized // no availability in requested area
```

Comment:
Why did you simplify the key? Was there anything off with it?

Reply:
I see, there were 2 keys for the same info and you chose to go with K3002. That's fine.